libs/kotlinlib/src/mill/kotlinlib/ksp/KspModule.scala

```diff
@@ -117,7 +117,7 @@ trait KspModule extends KotlinModule { outer =>
       "-no-stdlib",
       "-language-version",
       kotlinLanguageVersion()
-    )
+    ) ++ kotlincOptions()
```

Comment:
Should this be added at the start, so that in case something is duplicated, the `kspKotlincOptions` value overrides the other?